createst.py

```diff
@@ -142,7 +142,13 @@ def write_to_file(data):
     with open(test_yaml_path, "w+") as fp:
         fp.write("# *** Add configuration here ***\n\n")
         if not args["strictcsums"]:
-            fp.write("args:\n- -k none\n\n")
+            fp.write("args:\n- -k none\n")
+            if args["midstream"]:
+                fp.write("- --set stream.midstream=true\n")
```

Comment:
Q: What's the relationship of `strictcsums` and `midstream` for this check to be added here?

Reply:
They're not directly related, but I want to make sure that if one of `strictsums` has been added as an argument, we will not repeat the `args` line but will still add `midstream` setting if this option has been passed as well.
Wasn't able to think of a way of having them independent without repeating `args` ...

Reply:
I see. ONe of the better ways would be how it's done for `requires` by @TharushiJay : https://github.com/OISF/suricata-verify/pull/790/commits/411c482465c701b706abfb262e5683f13f1cd580
This would avoid the repetition of checks for individual args. :wink: